sspl_test/run_qa_test.sh

```diff
@@ -90,6 +91,21 @@ kill_mock_server()
 
 restore_cfg_services()
 {
+    # clear the dummy_service configurations made for
+    # alerts.os.test_service_monitor_sensor test
+    service_name=dummy_service.service
+    service_executable_code_des=/var/cortx/sspl/test
+    $sudo systemctl stop $service_name
+    $sudo systemctl disable $service_name
+    $sudo rm -rf $service_executable_code_des/dummy_service.py
+    $sudo rm -rf /etc/systemd/system/$service_name
+    $sudo systemctl daemon-reload
```

Comment:
Moved this reset for `alerts.os.test_service_monitor_sensor test`  since if we press CTRL+C, while tests the `dummy_service.service` would still get monitored after sspl-test complition.